<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>古籍可视化</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1171</width>
      <height>801</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>671</width>
        <height>721</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>0</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>40</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索结果</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>730</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>730</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>→</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>730</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>页码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>0</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>760</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>760</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>跳转</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>300</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
      <property name="intValue" stdset="0">
       <number>123123</number>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>420</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>530</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>450</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>检测阈值</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>560</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>识别阈值</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>420</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>420</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>530</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>530</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>400</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>510</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>111</width>
        <height>791</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>庄子</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>封面</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>序幕</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第一章</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第二章</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第三章</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第四章</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第五章</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第六章</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第七章</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第八章</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>书后</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWebEngineView" name="chart1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>541</width>
        <height>311</height>
       </rect>
      </property>
      <property name="url">
       <url>
        <string>file:///D:/Program Files (x86)/OCR/charts/word.html</string>
       </url>
      </property>
      <property name="zoomFactor">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QWebEngineView" name="chart2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>0</y>
        <width>501</width>
        <height>311</height>
       </rect>
      </property>
      <property name="url">
       <url>
        <string>file:///D:/Program Files (x86)/OCR/charts/pie.html</string>
       </url>
      </property>
     </widget>
     <widget class="QWebEngineView" name="chart3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>1101</width>
        <height>471</height>
       </rect>
      </property>
      <property name="url">
       <url>
        <string>file:///C:/Users/xujiadong/Desktop/新建文件夹/HTMLPage2.html</string>
       </url>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>阅读</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>视图</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1135</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu1_2">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu1_2"/>
   <addaction name="menu"/>
  </widget>
  <action name="action11111">
   <property name="text">
    <string>11111</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
